docker/files/entrypoint.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+import os
+from subprocess import call
+
+from dokang.harvesters import sphinx_rtd_config
+from dokang_pdf import PdfHarvester
+
+
+def sphinx_rtd_config_with_pdf():
+    return sphinx_rtd_config(pdf=PdfHarvester)
+
+
+CONFIG_ENV_VARS = (
+    ('DOKANG_UPLOADED_DOCS_TOKEN', None),  # Token is required
+    ('DOKANG_OPENSEARCH_NAME', 'My docs'),
+    ('DOKANG_OPENSEARCH_DESCRIPTION', 'My documents'),
+)
+
+
+def render_with_dokang_variables():
+
+    if not os.environ.get('DOKANG_UPLOADED_DOCS_TOKEN'):
+        raise ValueError('The upload token is required.')
+
+    with open('prod.template.ini', 'rt') as prod_template:
+        template = prod_template.read()
+
+    for var, default in CONFIG_ENV_VARS:
+        template = template.replace('__%s__' % var, os.environ.get(var, default))
```

Comment:
In fact, we could also use https://docs.python.org/3/library/string.html#template-strings which do the same job (I know its my idea to use replace() instead of Jinja...)